src/index.js

```diff
@@ -0,0 +1,170 @@
+}
+
+const changeLandscape = (temp) => {
+    let landscapeImage;
+    if (temp >= 80) {
+        landscapeImage = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    } else if (temp >= 70 && temp <= 79) {
+        landscapeImage = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (temp >= 60 && temp <=69) {
+        landscapeImage = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else {
+        landscapeImage = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲'
+    }
+    state.landscape.textContent = landscapeImage;
+}
+
+const increaseTemp = () => {
+    state.temp++;
+    state.tempValue.textContent = state.temp;
+    changeTempColor(state.temp);
+    changeLandscape(state.temp);
+}
+
+const decreaseTemp = () => {
+    state.temp--;
+    state.tempValue.textContent = state.temp;
+    changeTempColor(state.temp);
+    changeLandscape(state.temp);
+}
+
+const updateCity = () => {
+    state.headerCityName.textContent = state.cityNameInput.value;
+}
+
+const resetCity = () => {
+    state.cityNameInput.value = state.cityNameInput.placeholder;
+    updateCity()
+    updateTempByCity()
+    state.cityNameInput.value = "";
+}
+
+const getLatitudeLongitude = (place) => {
+    return axios.get('https://weather-report-proxy-server-vaa7.onrender.com/location', {
+        params: {q: place}
+    })
+    .then( response => {
+        const lat = response.data[0].lat;
+        const lon = response.data[0].lon;
+        return {lat, lon}
+    })
+    .catch( error => {
+        console.log(error);
+    })
+}
+
+const convertKToF = k => Math.round((k-273.15) * (9/5) + 32)
+
+const getLocationWeather = (city) => {
+    return getLatitudeLongitude(city)
+    .then( response => {
+        return axios.get('https://weather-report-proxy-server-vaa7.onrender.com/weather', {
+            params: {lat: response.lat, 
+                lon: response.lon
+            }
+        })
+        .then( response => {
+            state.temp = convertKToF(response.data.main.temp);
+            return state.temp;
+        })
+        .catch( error => {
+            console.log(error);
+        })
+    })
+}
+
+const updateTempByCity = () => {
+    getLocationWeather(state.cityNameInput.value)
+    .then(temp => {
+        state.tempValue.textContent = temp;
+        changeTempColor(temp);
+        changeLandscape(temp);
+    });
+}
+
+const changeSky = () => {
+    let skyGraphic;
+    let gardenContentColor = 'garden__content ';
+
+    if (state.skySelect.value === 'sunny') {
+        skyGraphic = "☁️  ☁️  ☁️  ☀️  ☁️  ☁️";
+        gardenContentColor += 'sunny'
+    } else if (state.skySelect.value === 'cloudy') {
+        skyGraphic = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+        gardenContentColor += 'cloudy';
+    } else if (state.skySelect.value === 'rainy') {
+        skyGraphic = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+        gardenContentColor += 'rainy';
+    } else {
+        skyGraphic = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+        gardenContentColor += 'snowy';
+    }
+    state.sky.textContent = skyGraphic
+    state.gardenContent.setAttribute('class', gardenContentColor)
+}
```

Comment:
Same as above! Let's turn this into an object, so we can cut down on all these if/else statements